### PR TITLE
[New Rule] UAC Bypass via Mocking Windir

### PR DESCRIPTION
**Pull request by Samirbous**
## Issues

Resolves https://github.com/elastic/detection-rules/issues/408

## Contributor checklist

- Have you signed the [contributor license agreement](https://www.elastic.co/contributor-agreement)?
- Have you followed the [contributor guidelines](https://github.com/elastic/detection-rules/blob/main/CONTRIBUTING.md)?

### CODE REVIEW

File: rules/windows/privilege_escalation_uac_bypass_mock_windir.toml [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+author = ["Elastic"]
+description = """
+Identifies User Account Control (UAC) bypass via Mocking Microsoft Trusted Windows directory. Attackers bypass UAC to
+stealthily execute code with elevated permissions.
+"""
+from = "now-9m"
+index = ["winlogbeat-*", "logs-endpoint.events.*"]
+language = "eql"
+license = "Elastic License"
+name = "UAC Bypass Attempt via Mocking Windows Directory"
```
Comment (bm11100) [inline rules/windows/privilege_escalation_uac_bypass_mock_windir.toml:17]: I'm not a huge fan of `mocking`. Maybe we use something like mimic, impersonate, imitate, masquerade, etc., for this and the description. Maybe `UAC Bypass Attempt via Windows Directory Masquerading`
```suggestion
name = "UAC Bypass Attempt via Windows Directory Masquerading"
```

**Review by bm11100 — APPROVED** [review]

File: rules/windows/privilege_escalation_uac_bypass_mock_windir.toml [hunk 2]
```diff
@@ -0,0 +1,42 @@
+[metadata]
+creation_date = "2020/10/26"
+ecs_version = ["1.6.0"]
+maturity = "production"
+updated_date = "2020/10/26"
+
+[rule]
+author = ["Elastic"]
+description = """
+Identifies an attempt to bypass User Account Control (UAC) by masquerading as a Microsoft Trusted Windows directory. Attackers may bypass UAC to
```
Comment (brokensound77) [inline rules/windows/privilege_escalation_uac_bypass_mock_windir.toml:10]:
```suggestion
Identifies an attempt to bypass User Account Control (UAC) by masquerading as a Microsoft trusted Windows directory. Attackers may bypass UAC to
```

File: rules/windows/privilege_escalation_uac_bypass_mock_windir.toml [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+references = ["https://medium.com/tenable-techblog/uac-bypass-by-mocking-trusted-directories-24a96675f6e"]
+risk_score = 71
+rule_id = "290aca65-e94d-403b-ba0f-62f320e63f51"
+severity = "high"
+tags = ["Elastic", "Host", "Windows", "Threat Detection", "Privilege Escalation"]
+type = "eql"
+
+query = '''
+process where event.type in ("start", "process_started") and
+ wildcard(process.args, "C:\\Windows \\system32\\*.exe", "C:\\Windows \\SysWOW64\\*.exe")
```
Comment (brokensound77) [inline rules/windows/privilege_escalation_uac_bypass_mock_windir.toml:27]:
```suggestion
process where event.type in ("start", "process_started") and
  process.args : ("C:\\Windows \\system32\\*.exe", "C:\\Windows \\SysWOW64\\*.exe")
```

**Review by dstepanic17 — APPROVED** [review]
LGTM!

**Pull request merged**